layers/+lang/purescript/packages.el

```diff
@@ -39,6 +39,8 @@
     (progn
       (add-hook 'purescript-mode-hook 'turn-on-purescript-indentation)
       (add-hook 'purescript-mode-hook 'purescript-decl-scan-mode)
+      (spacemacs/declare-prefix "mg" "jump")
```

Comment:
"mg" is generally "goto" in other layers, I think you missed out on "mh" = "help" too

Reply:
We don't have any 'help' commands in the purescript layer at the moment. Should I define it anyway in case we add some?

Reply:
There was one (`psc-ide-show-type`), but I moved it from `mht` to `mt` in this commit. I could also move it back, and give `mh` the 'help' prefix.

Reply:
Oh right, I didn't notice that got shifted as well. There's a "mt" prefix = "tests" convention in other language layers although I'm not sure how applicable that is in Purescript. 
http://spacemacs.org/doc/CONVENTIONS#tests

It would also be helpful to have a `"mhh"` binding for "K" in evil normal state to bring up language-specific documentation, again not sure what that maps to in the Purescript packages.

Reply:
Ah, thanks for the link. I moved "mt" back to "mht" because that seems like the pretty standard location for that command. 

The purescript compiler does have a documentation generator ala javadoc or something, but I don't believe the emacs plugins have any way to show that.